src/domains/global/utils.ts

```diff
@@ -0,0 +1,11 @@
+const allowedReferrerDomains = [
+  "",
+  "https://www.google.com/",
+  "https://duckduckgo.com/",
+  "https://www.reddit.com/",
+]
+
+export const isAllowedReferrer = (referrer: string) => allowedReferrerDomains.includes(referrer)
+  || referrer.endsWith(".my-netdata.io/")
+  || referrer.startsWith("https://github.com/")
+  || referrer.endsWith("netdata.cloud/")
```

Comment:
Should we add a 
```
|| referrer.startsWith("https://app.netdata.cloud/")
```

As i do also see some referrer strings like this:

https://app.netdata.cloud/spaces/andrewm4894/rooms/general/overview

Which could be very useful.